.travis.yml

```diff
@@ -1,8 +1,7 @@
 env:
   global:
-    - MAKEFLAGS="-j 4"
```

Comment:
Cargo sets CMake to use the number of threads equal to whatever the system has, so this does nothing in the new build. OpenCV is being built by Cargo now, rather than from a Makefile.